doc_classes/LuaAPI.xml

```diff
@@ -22,23 +23,26 @@
 		</method>
 		<method name="bind_libraries">
 			<return type="void" />
-			<param index="0" name="Array" type="Array" />
+			<param index="0" name="Array" type="Godot.Collections.Array" />
```

Comment:
I think this would mess up auto type completion for GDScript

Reply:
Will it? Hmmn. As it is, it picks an Array, and because I'm also using the System namespace, it causes an Ambiguous reference. That's why I was trying to nail it down.

Reply:
I believe the official modules use just `Array`, probably safest to follow suite

Reply:
Okay, can do!